src/stravalib/model.py

```diff
@@ -187,14 +188,14 @@ def check_valid_location(
 
     Parameters
     ----------
-    location : list of floats
-        Either a list of x,y floating point values or strings or None
+    location : Sequence of floats
```

Comment:
so how do we update the docstrings? can we leave them as is given this is just a typing update rather than an actual structural update?

Reply:
Good question. If we want to be 100% _correct_ in our documentation, I'd go for Sequence (as we technically cannot guarantee it will be lists (vs., e.g., tuples) - that depends on how Pydantic parses the raw data). However, it looks like Pydantic currently parses json arrays into a list when the field type is a Sequence. So, the pragmatic approach would be to keep the docstring type as list, and only update the docstrings whenever the Pydantic implementation changes (which may never happen).